runtime-modules/content/src/permissions/mod.rs

```diff
@@ -299,6 +299,50 @@ pub fn ensure_actor_authorized_to_manage_nft<T: Trait>(
     Ok(())
 }
 
+/// Ensure actor can destroy nft
+pub fn ensure_actor_authorized_to_destroy_nft<T: Trait>(
```

Comment:
This seems virtually identical to `ensure_actor_authorized_to_manage_nft`, I think perhaps you made this because you thought you needed to block `NftOwner::Member`, but you actually don't :D

Reply:
That right :) will just revert to using `ensure_actor_authorized_to_manage_nft` which has the correct permission check for this action.